src/resources/lang/UltraStar-Manager.de.ts

```diff
@@ -28,14 +28,14 @@
         <translation>Zum Editieren hier klicken.</translation>
     </message>
     <message>
-        <location filename="../../songtree/QUSongItem.cpp" line="116"/>
+        <location filename="../../songtree/QUSongItem.cpp" line="113"/>
         <source>Unsupported Tags:
 #</source>
         <translation>Nicht unterstützte Tags:
 #</translation>
     </message>
     <message>
-        <location filename="../../main.cpp" line="135"/>
+        <location filename="../../main.cpp" line="136"/>
```

Comment:
Similarly here, where do these massive changes come from? `main.cpp` wasn't even changed, right?

Reply:
1. No main.cpp wasn't changed. One of Monty's tips is about swisscharts, so I had to update all translations (`lupdate UltraStar-Manager.pro` and `lrelease UltraStar-Manager.pro` (see `lupdate-all.cmd` and `lrelease-all.cmd` in the src folder, for Windows).